python/ray/data/_internal/stats.py

```diff
@@ -549,6 +600,22 @@ def _run_update_loop():
 
     # Execution methods
 
+    def _aggregate_per_node_metrics(
+        self, op_metrics: List[OpRuntimeMetrics]
+    ) -> Optional[Mapping[str, Mapping[str, Union[int, float]]]]:
```

Comment:
Could you maybe add a docstring explaning what the returns represents? Had some difficulty parsing the code to understand this